src/sentry/core/endpoints/organization_details.py

```diff
@@ -558,6 +567,85 @@ def save(self, **kwargs):
         if trusted_relay_info is not None:
             self.save_trusted_relays(trusted_relay_info, changed_data, org)
 
+        if "hasGranularReplayPermissions" in data or "replayAccessMembers" in data:
+            if not features.has("organizations:granular-replay-permissions", org):
+                raise serializers.ValidationError(
+                    {
+                        "hasGranularReplayPermissions": "This feature is not enabled for your organization."
+                    }
+                )
```

Comment:
You could have a `validate_*` method for these fields in the serializer that applies validation *before* the save process is started.